### PR TITLE
Directory configuration implementation

### PR DESCRIPTION
**Pull request by godwinamila**

### CODE REVIEW

File: jaggeryapps/dashboard/directories/add_finish.jag [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+var webService = new ws.WSRequest();
+var i18n = session.get("i18n");
+
+function addUserStore(){
+
+    try{
+        var name = request.getParameter('name');
+        var url = request.getParameter('url');
+
+        var domain="is-wso2.com"; //TODO remove magic words
```
Comment (harsha1979) [inline jaggeryapps/dashboard/directories/add_finish.jag]: Is this forgot or, aware ?

File: jaggeryapps/dashboard/directories/delete_finish.jag [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+var log = new Log();
+include("../controllers/wsUtil.jag");
+var util = require("../util/utility.jag");
+
+var serverUrl = util.getServerUrl();
+serverUrl = serverUrl + "/UserStoreConfigAdminService";
+var ws = require('ws');
+var webService = new ws.WSRequest();
+var i18n = session.get("i18n");
```
Comment (harsha1979) [inline jaggeryapps/dashboard/directories/delete_finish.jag]: var i18n = session.get("i18n"); No usages in any places... is that expected to use it later

File: jaggeryapps/dashboard/directories/get-directory.jag [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+var i18n = session.get("i18n");
+
+var domain = request.getParameter("domain");
+
+getDirectory();
+function getDirectory(){
+    try{
+        webService = new ws.WSRequest();
+        var result = callWSReturnJSON(webService, serverUrl, "urn:getSecondaryRealmConfigurations", null);
+        //return result;
```
Comment (harsha1979) [inline jaggeryapps/dashboard/directories/get-directory.jag]: remove this

File: jaggeryapps/dashboard/directories/get-directory.jag [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        print(result);
+    }catch(e){
+        print(e);
```
Comment (harsha1979) [inline jaggeryapps/dashboard/directories/get-directory.jag]: why we are doing both , log and print

File: jaggeryapps/dashboard/directories/get-directory-list.jag [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+var ws = require('ws');
+var webService = new ws.WSRequest();
+var i18n = session.get("i18n");
+
+getDirectories();
+function getDirectories(){
+    try{
+        webService = new ws.WSRequest();
+        var result = callWSReturnJSON(webService, serverUrl, "urn:getSecondaryRealmConfigurations", null);
+        //return result;
```
Comment (harsha1979) [inline jaggeryapps/dashboard/directories/get-directory-list.jag]: remove this

File: jaggeryapps/dashboard/directories/add_finish.jag (hunk 1 above)
Comment (godwinamila) [inline jaggeryapps/dashboard/directories/add_finish.jag]: Fixed

File: jaggeryapps/dashboard/directories/delete_finish.jag (hunk 2 above)
Comment (godwinamila) [inline jaggeryapps/dashboard/directories/delete_finish.jag]: fixed

File: jaggeryapps/dashboard/directories/get-directory-list.jag (hunk 5 above)
Comment (godwinamila) [inline jaggeryapps/dashboard/directories/get-directory-list.jag]: fixed

**Pull request merged**